Enable VDU0 and LVDS panel in dts for tf307-bm-s-d board

From: Miniakhmetov Vitaly <vitalii.miniakhmetov@t-platforms.ru>

diff --git a/arch/arm64/boot/dts/baikal/edelweiss-tf307-mb-s-d.dts b/arch/arm64/boot/dts/baikal/edelweiss-tf307-mb-s-d.dts
index 6dfc42b94212..7937b6aea4ea 100644
--- a/arch/arm64/boot/dts/baikal/edelweiss-tf307-mb-s-d.dts
+++ b/arch/arm64/boot/dts/baikal/edelweiss-tf307-mb-s-d.dts
@@ -14,7 +14,7 @@ / {
 	panel {
 		/* In order to utilize LVDS LCD panel,
 		   set this to "okay" along with &vdu0 status property (see below). */
-		status = "disabled";
+		status = "okay";
 		compatible = "panel-lvds";
 		width-mm = <223>;
 		height-mm = <125>;
@@ -60,7 +60,7 @@ &gmac1 {
 &vdu0 {
 	/* In order to utilize LVDS LCD panel,
 	   set this to "okay" along with panel status property (see above). */
-	status = "disabled";
+	status = "okay";
 	port {
 		vdu_lvds0_pads: endpoint@0 {
 			remote-endpoint = <&panel0_lvds0>;
